app/services/katello/organization_creator.rb

```diff
@@ -0,0 +1,78 @@
+module Katello
+  class OrganizationCreator
+    attr_reader :library_view, :library_environment, :library_cvv
+
+    def self.seed_all_organizations
+      User.as_anonymous_admin do
+        Organization.not_created_in_katello.each do |org|
+          self.new(org).seed!
+        end
+      end
+    end
+
+    def self.create_all_organizations
+      User.as_anonymous_admin do
+        Organization.not_created_in_katello.each do |org|
+          ForemanTasks.sync_task(::Actions::Katello::Organization::Create, org)
+        end
+      end
+    end
+
+    def initialize(organization)
+      @organization = organization
+    end
+
+    def seed!
+      @organization.setup_label_from_name
```

Comment:
is this a good time to correct my grammar pet peeve and rename this method? :yum: 
```suggestion
      @organization.set_up_label_from_name
```

Reply:
I like it. But the module that defines this method is used in several models so I'd prefer to leave any change out of this PR :-/